sql-statements/sql-statement-foreign-key.md

```diff
@@ -0,0 +1,255 @@
+    [identifier] (col_name, ...)
+    REFERENCES tbl_name (col_name,...)
+    [ON DELETE reference_option]
+    [ON UPDATE reference_option]
+```
+
+The foreign key can be self-referencing, that is, referencing the same table. When you add a foreign key constraint to a table using `ALTER TABLE`, you need to first create an index on the parent table column that the foreign key references.
+
+## Delete a foreign key constraint
+
+To delete a foreign key constraint, you can use the following `ALTER TABLE` statement:
+
+```sql
+ALTER TABLE table_name DROP FOREIGN KEY fk_identifier;
+```
+
+If the foreign key constraint is named when it is created, you can reference the name to delete the foreign key constraint. Otherwise, the constraint name is generated automatically. You can use `SHOW CREATE TABLE` to view the foreign key name:
+
+```sql
+mysql> SHOW CREATE TABLE child\G
+*************************** 1. row ***************************
+       Table: child
+Create Table: CREATE TABLE `child` (
+  `id` int(11) DEFAULT NULL,
+  `pid` int(11) DEFAULT NULL,
+  KEY `idx_pid` (`pid`),
+  CONSTRAINT `fk_1` FOREIGN KEY (`pid`) REFERENCES `test`.`parent` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
+```
+
+## Foreign key constraint check
+
+Whether to enable the foreign key constraint check is controlled by the system variable [`foreign_key_checks`](/system-variables.md#foreign_key_checks). By default, this variable value is set to `ON`, meaning that the foreign key constraint check is enabled. This variable has two scopes: `GLOBAL` and `SESSION`. Keeping this variable enabled can ensure the integrity of foreign key reference relationships.
+
+The effect of disabling [`foreign_key_checks`](/system-variables.md#foreign_key_checks) is as follows:
+
+- When deleting a parent table referenced by a foreign key, the deletion can succeed only when the foreign key constraint check is disabled.
+- When importing data to a database, the order of creating tables might be different from the foreign key dependency order, which might cause the creation of tables to fail. Only when the foreign key constraint check is disabled can the tables be created successfully. In addition, when importing data, disabling the foreign key constraint check can also speed up data import.
+- When importing data to a database, if the data of the child table is imported first, an error will be reported. Only when the foreign key constraint check is disabled can the data of the child table be imported successfully.
+- When executing `ALTER TABLE` operations related to foreign keys, the foreign key constraint check must be enabled to execute successfully.
+
+When the foreign key constraint check is disabled, the foreign key constraint check and reference operation are not executed, except for the following scenarios:
+
+- When executing `ALTER TABLE`, if the definition of the foreign key is incorrect, an error is still reported.
```

Comment:
Inconsistent with the Chinese. Please confirm. If we follow the Chinese, the translation should be:
> If the execution of  `ALTER TABLE` might result in wrong definition of the foreign key, an error is still reported during the execution.

Reply:
I already change the Chinese version, and this is lgtm.